examples/research_projects/codeparrot/README.md

```diff
@@ -125,7 +127,8 @@ python scripts/human_eval.py --model_ckpt lvwerra/codeparrot \
 --do_sample True \
 --temperature 0.2 \
 --top_p 0.95 \
---n_samples=200
+--n_samples=200 \
+--HF_ALLOW_CODE_EVAL="1"
```

Comment:
This was set to 0 by default on purpose, but I agree we can make the flag more visible. 

```suggestion
--HF_ALLOW_CODE_EVAL="0"
```

The reason I would like to avoid being able to run the code by default is that this metric executes the generated snippets on the machine. The generations could be harmful so it is important that the user is aware of the risks. But I agree that getting error only after the generation is complete is annoying as this can take a long time. We could do a quick test at the beginning e.g.:

```python
try:
    _ = metric.compute(references=[''], predictions=[['']])
except Exception as e:
    print("Code evaluation not enabled. Read the warning below carefully and then use `--HF_ALLOW_CODE_EVAL '1'` flag to enable code evaluation.")
    raise e
```

Reply:
Makes total sense to give it as 0 so that they don't execute dangerous code by accident. I'll add this try except to the beginning so people don't spend their wheels!